packages/sfc-playground/src/VersionSelect.vue

```diff
@@ -74,8 +74,8 @@ onMounted(() => {
 
     <ul class="versions" :class="{ expanded }">
       <li v-if="!versions"><a>loading versions...</a></li>
-      <li v-for="version of versions">
-        <a @click="setVersion(version)">v{{ version }}</a>
+      <li v-for="ver of versions" :class="{ active:  (pkg === 'vue' ? 'v' + ver : ver) === version }">
```

Comment:
Let's remove `v` prefix in https://github.com/vuejs/core/blob/2ffe3d5b3e953b63d4743b1e2bc242d50916b545/packages/sfc-playground/src/Header.vue#L27

Reply:
Done.